include/zephyr/drivers/timer/nrf_rtc_timer.h

```diff
@@ -65,6 +65,22 @@ uint64_t z_nrf_rtc_timer_read(void);
  */
 uint32_t z_nrf_rtc_timer_compare_evt_address_get(int32_t chan);
 
+/** @brief Configure publishing of the COMPARE event to (D)PPI.
+ *
+ * @param chan Channel ID between 1 and CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT.
+ *
+ * @param ppi (D)PPI channel to publish to.
+ */
+void z_nrf_rtc_timer_compare_evt_publish_set(int32_t chan, uint32_t ppi);
+
+/** @brief Clear COMPARE event publishing configuration.
+ *
+ * @param chan Channel ID that was previously configured.
+ *
+ * @param ppi (D)PPI channel to which publishing is to be cleared.
+ */
+void z_nrf_rtc_timer_compare_evt_publish_clear(int32_t chan, uint32_t ppi);
```

Comment:
I don't quite see the benefit of introducing these functions. Is it that complicated to use what is already available:
```
nrfx_gppi_event_endpoint_setup(ppi, evt_addr);
```
and
```
nrfx_gppi_event_endpoint_clear(ppi, evt_addr);
```
where `evt_addr = z_nrf_rtc_timer_compare_evt_address_get(chan)`?

Reply:
Yes, nrfx_gppi fits my needs. I missed the presence of such a helper. Thanks for the tip.

The changes to the nrf_rtc_timer are not needed then. I am closing this pr.